=begin pod

=TITLE role CX::Succeed

=SUBTITLE Succeed control exception

    role CX::Succeed does X::Control { }

A L<control exception|/language/exceptions#Control_exceptions> thrown when
C<succeed> is called from a C<when> or C<default> block; since Rakudo 2019.03,
C<throw>ing an C<X::Control> object raises a control exception which is caught
by the L<CONTROL phaser|/language/phasers#CONTROL> instead of
L<CATCH|/language/phasers#CATCH>.

=head1 Methods

=head2 method message

Defined as:

    method message()

Returns "<next control exception>"

=end pod
